src/sage/matrix/special.py

```diff
@@ -2407,6 +2407,32 @@ def companion_matrix(poly, format='right'):
     return M
 
 
+def random_subset(n, k):
+    """
+    Return a random subset of range(n) of size k.
+
+    The distribution law is uniform.
```

Comment:
Here you use uniform distribution but the original method used a beta distribution with parameters `[1.6, 4.3]`. ..